test/compiler_test.rb

```diff
@@ -40,6 +40,10 @@ def test_freshness
     assert !Webpacker.compiler.fresh?
   end
 
+  def test_compile
+    assert !Webpacker.compiler.compile
```

Comment:
@haberbyte how does this test work to assert that the `compiler.compile` command returns false?

Reply:
Hm, good question, this is actually just the minimal failing test i came up with that would break if you miss the `.to_s` fix. I also think it is weird that it returns false, but did not think much about it.

After some more investigation, it looks like it returns `false` because the command to run in the `Open3.capture3` block can not find "webpack". Stepping into this with a breakpoint i get `Compilation failed: error Command "webpack" not found.`

If this is really intended, i guess hiding that in my test in not good. Not sure how to go about this, but maybe stubbing the webpack command in the test suite would work.